components/collective-navbar/ActionsMenu.js

```diff
@@ -132,11 +152,92 @@ export const getContributeRoute = collective => {
   return { route, params };
 };
 
-const CollectiveNavbarActionsMenu = ({ collective, callsToAction, hiddenActionForNonMobile }) => {
+const getSecondCTA = (enabledCTAs, hasRequestGrant, mainAction) => {
+  if (!hasRequestGrant) {
+    return enabledCTAs;
+  } else {
+    return difference(enabledCTAs, mainAction);
```

Comment:
:no_entry_sign: I don't think that this one is working properly. Before this PR, I have two CTAs for `/captainfact_io`: contact and submit expense. After this PR, the contact button is gone and I only end up with the first one.

Logging the different steps:
> enabledCTAs: ["hasContact", "hasSubmitExpense"]
> mainAction ["hasSubmitExpense"]
> secondAction ["hasContact", "hasSubmitExpense"]